src/test/javascript/spec/component/assessment-dashboard/exercise-assessment-dashboard.component.spec.ts

```diff
@@ -343,32 +345,31 @@ describe('ExerciseAssessmentDashboardComponent', () => {
     });
 
     it('should set exam and stats properties', () => {
-        expect(comp.exam).toBeUndefined();
+        expect(comp.exam).toBe(undefined);
 
         comp.loadAll();
-
-        expect(comp.exercise).toBeDefined();
+        expect(comp.exercise.id).toBe(modelingExercise.id);
         expect(comp.exam).toEqual(exam);
-        expect(comp.exam?.numberOfCorrectionRoundsInExam).toEqual(numberOfAssessmentsOfCorrectionRounds.length);
-        expect(comp.numberOfAssessmentsOfCorrectionRounds).toEqual(numberOfAssessmentsOfCorrectionRounds);
+        expect(comp.exam?.numberOfCorrectionRoundsInExam).toBe(numberOfAssessmentsOfCorrectionRounds.length);
+        expect(comp.numberOfAssessmentsOfCorrectionRounds).toBe(numberOfAssessmentsOfCorrectionRounds);
```

Comment:
As both objects are arrays containing more or less complex objects, I would go for `toEqual`.
```suggestion
        expect(comp.numberOfAssessmentsOfCorrectionRounds).toEqual(numberOfAssessmentsOfCorrectionRounds);
```